libwarpaffine/document_info.h

```diff
@@ -142,5 +142,7 @@ struct DeskewDocumentInfo
 
     double          z_scaling{ std::numeric_limits<double>::quiet_NaN() };  ///< The size of one pixel in z-direction in units of micro-meters.
     double          xy_scaling{ std::numeric_limits<double>::quiet_NaN() }; ///< The size of one pixel in x- or y-direction in units of micro-meters.
+
+    const double angle_in_radians = 60.0 / 180.0 * 3.14159265358979323846;
```

Comment:
I reckon the plan is - to put the "adjustable parameter" here at some point, i.e. later on determine the angle and make this a variable then.
Maybe adding some documentation would be nice. Also - is there a better name (than "angle") for this?